docs/cli/manifest_config.mdx

```diff
@@ -111,7 +111,7 @@ Additionally, the latest version of `dockerd` recognizes OCI manifests and does
 ## Push / Pull config
 
 As mentioned in the beginning of this doc, the manifest config is not used by `oras` and is not worth pulling in most scenarios.
-However, it is still possible to push pullable config with the `oras` CLI by leveraging [Manifest Annotations](./manifest_annotations).
+However, it is still possible to push pullable config with the `oras` CLI by leveraging [Manifest Annotations](/cli/manifest_annotations).
```

Comment:
```suggestion
However, it is still possible to push pullable config with the `oras` CLI by leveraging [Manifest Annotations](./manifest_annotations).
```
Since the file is in the same folder, we just need to link it using `./`. Adding `/cli/` changes the URL to `https://oras.land/CLI/manifest_annotations`, which is invalid link. The correct link after changes would be `https://oras.land/docs/cli/manifest_annotations`. All the documentation in Docusaurus are present in `docs` folder, hence it is refelcted in the URL.